src/list.rs

```diff
@@ -27,6 +27,7 @@ pub struct ListItem {
 #[derive(Clone, PartialEq)]
 pub struct TextColumn {
     pub text: ViewString,
+    pub display: Option<String>,
```

Comment:
This change should be mentioned in the CHANGELOG, as it is the change from the users' perspective.

Reply:
Done